app/helpers/interruption_screen_helper.rb

```diff
@@ -1,5 +1,6 @@
 module InterruptionScreenHelper
   def display_informative_text(informative_text, log)
+    return informative_text if informative_text.is_a? String
```

Comment:
let's just add one test to the spec for this helper that it now also accepts a string